cmd/version.go

```diff
@@ -25,6 +68,18 @@ func versionCommand(l logger, host string) *cli.Command {
 		RunE: func(c *cli.Command, args []string) error {
 			l.Printf(fmt.Sprintf("client: %s-%s", coloredNotice(config.Version), config.BuildCommit))
 
+			currentV, err := version.NewVersion(config.Version)
+			if err != nil {
+				return errors.Wrapf(err, "failed to parse current version")
+			}
+			latestV, err := version.NewVersion(checkLatestVersion())
+			if err != nil {
+				return errors.Wrapf(err, "failed to parse latest version")
```

Comment:
when `checkLatestVersion` will return an empty string at the time of error, we should not return error here, simply ignore. We want this check for update flow non-obstructive